docs/topics/scale.md

```diff
@@ -1,16 +1,14 @@
 # Scaling Kubernetes Clusters
 
-## Install Pre-requisites
+## Prerequisites
 
-All the commands in this guide require both the Azure CLI and `aks-engine`. Follow the [installation instructions to download aks-engine before continuing](../aksengine.md#install-aks-engine) or [compile from source](../aksengine.md#build-from-source).
+All the commands in this guide require both the Azure CLI and `aks-engine`. Follow the [quickstart guide](../tutorials/quickstart.md) before continuing.
 
-For installation instructions see [the Azure CLI GitHub repository](https://github.com/Azure/azure-cli#installation) for the latest release.
-
-This guide assumes you already have deployed a cluster using acs engine. For more details on how to do that see [deploy](./deploy.md).
+This guide assumes you already have deployed a cluster using aks-engine. For more details on how to do that see [deploy](../tutorials/deploy.md).
 
 ## Scale
 
-After a cluster has been deployed using acs engine the cluster can be interacted further by using the scale command. The scale command can add more nodes to an existing node pool or remove them. Nodes will always be added or removed from the end of the agent pool. Nodes will be cordoned and drained before deletion.
+The `aks-engine scale` command can increase or decrease the number of nodes in an existing agent pool in an `aks-engine` Kubernetes cluster. Nodes will always be added or removed from the end of the agent pool. Nodes will be cordoned and drained before deletion.
 
 This guide will assume you have a cluster deployed and the output for the deployed cluster is stored at _output/mycluster. It will also assume there is a node pool named "agentpool1" in your cluster. AKS Engine will default to storing the output at ./_output/dns-prefix from where the aks-engine command was ran.
```

Comment:
while we're here let's issue a grammar citation for "was ran", and correct it to "was run". :)

Reply:
Also, let's change `./_output/dns-prefix` to `./_output/<dnsPrefix>` to more clearly indicate we're talking about the variable `dnsPrefix` that's in the api model input.

Reply:
Let's modify the below paragraph. From:

```
This command will look the deployment directory to find info about the cluster currently deployed. Then it will generate and deploy a template deployment to update the cluster and add the new nodes. When it is done it will update the cluster definition in the deployment directory's apimodel.json to reflect the new node count.```

To:

```
This command will re-use the apimodel.json inside the output directory as input for a new ARM template deployment that will execute the scaling operation against the desired agent pool. When it the scaling operation is done it will update the cluster definition in that same apimodel.json file to reflect the new node count (and thus the updated, current cluster configuration.
```

Something like that

Reply:
Also let's change the parameters table so that the params are expressed using the `--param-name` (i.e., the `--` prefix), for clarity